lib/options.php

```diff
@@ -11,7 +11,7 @@
     define('TWOFA_WINDOW', 2);
 }
 
-// Keep tokens in the blacklist for this amount of seconds
-if (!defined('TWOFA_BLACKLIST_DURATION')) {
-    define('TWOFA_BLACKLIST_DURATION', 60*5);
+// Keep tokens in the denylist for this amount of seconds
+if (!defined('TWOFA_DENYLIST_DURATION')) {
```

Comment:
Worth noting that this will be a breaking change for any site that defines the `TWOFA_BLACKLIST_DURATION`. I think this is only used on gds-blogs, which doesn't do so. But we might want to check.